AsyncDisplayKit/ASMapNode.mm

```diff
@@ -160,7 +162,11 @@ - (void)takeSnapshot
   if (!_snapshotter) {
     [self setUpSnapshotter];
   }
-  [_snapshotter cancel];
+  
+  if (_snapshotter.isLoading) {
```

Comment:
@appleguy The only thing that this may have a negative side affect on is if someone is changing the ASMapNode options rapidly or on slow connections. With this implementation the snapshotter simply rejects the new change, rather than aborting the current change and replacing it with this new one. How have you found it in practice?